crest/Assets/Crest/Crest/Scripts/Collision/QueryFlow.cs

```diff
@@ -21,23 +21,26 @@ public class QueryFlow : QueryBase
 
         protected override void OnEnable()
         {
-            Debug.Assert(Instance == null);
             Instance = this;
 
             base.OnEnable();
         }
 
         protected override void OnDisable()
         {
-            Instance = null;
+            // If we set this to null, it breaks exiting play mode, as OnDisable is called but no matching call to OnEnable :/
+            //Instance = null;
```

Comment:
Same as previous comment. Might be worth tagging these if it happens in multiple place.